.changeset/hot-pumas-pump.md

```diff
@@ -0,0 +1,5 @@
+---
+'create-svelte': minor
```

Comment:
is this a major because vitest also requires node18 ? ultimately we are going to do more changes to the templates in create-svelte that cause a major so not sure it matters much apart from where in the changelog its listed

Reply:
It's a good question. We have lots of things causing a major elsewhere but nothing on `create-svelte` yet. If we want to do a major this wouldn't be a bad PR to do it in. I don't know if we're worried about that causing issues on StackBlitz, etc. or anywhere like that